### PR TITLE
Add apptainer support

### PR DESCRIPTION
**Pull request by johnbradley**
Adds apptainer definition and config to automatically build the apptainer image on release. The apptainer image is based on the docker image.

Part of #27

### CODE REVIEW

**Comment by johnbradley** [comment]
Apptainer copies the labels from the upstream image so no extra config was needed.

File: apptainer.def [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Bootstrap: docker
+From: ghcr.io/imageomics/pybioclip:{{ PYBIOCLIP_VERSION }}
+
+%arguments
+    PYBIOCLIP_VERSION=
+
+%runscript
+   exec bash
```
Comment (hlapp) [inline apptainer.def:8]: I'm using `exec "$@"` elsewhere, but that's for a container that has a defined entry point defined in the Dockerfile (a Jupyter container). Not sure how to assess whether that would be better or not.

**Review by hlapp — APPROVED** [review]
See one inline comment for consideration. Otherwise looks 👍

File: apptainer.def (hunk 1 above)
Comment (johnbradley) [reply apptainer.def:8]: I tried this out and it does work better. I'm going to make this change.

**Pull request merged**